### PR TITLE
disable mobile services by default for managed clusters

### PR DESCRIPTION
**Pull request by wei-lee**

### CODE REVIEW

**Comment by wei-lee** [comment]
ping @grdryn @odra

**Comment by odra** [comment]
there is a `datasync` var as well

**Comment by odra** [comment]
let me try it in a cluster

**Comment by odra** [comment]
The vars ddn't work as all components are still being installed but no related to this pr - those checks may need to be improved

**Comment by odra** [comment]
ok I see there are no usage of those vars in the install_mobile_services playbook which is why it didn't work - could you please add them there?

**Comment by wei-lee** [comment]
@odra right, yes. will do

**Comment by wei-lee** [comment]
@odra mind taking another look?

**Comment by odra** [comment]
giving it a try again

File: playbooks/install_mobile_services.yml [hunk 1]
```diff
@@ -13,24 +13,28 @@
...
 
     - name: Install mobile developer console (mdc)
       include_role:
         name: mdc
       vars:
         openshift_master_url: "{{ hostvars['EVAL_VARS']['openshift_master_url'] | replace('https://', '') }}"
       tags: ['mdc']
+      when: mdc | default(true) | bool
 
     - name: User rhsso configuration
```
Comment (odra) [inline playbooks/install_mobile_services.yml]: this one needs a mdc check as well

Comment (wei-lee) [reply playbooks/install_mobile_services.yml]: done

**Comment by wei-lee** [comment]
@david-martin do you mind reviewing this PR as well and see if I am doing the feature flags correctly? Thanks.

File: playbooks/install_mobile_manifest.yml [hunk 2]
```diff
@@ -126,26 +129,31 @@
...
+        - run_master_tasks | default(true) | bool 
+        - mdc | default(true) | bool
     
     - name: restart openshift master services
       shell: "/usr/local/bin/master-restart {{ item }}"
       become: yes
       with_items:
         - api
         - controllers
+      when: mdc | default(true) | bool
```
Comment (david-martin) [inline playbooks/install_mobile_manifest.yml]: Should `run_master_tasks | default(true) | bool` also be satisfied here too?

**Pull request merged**